modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/Layout/LayoutScreen/ModalAddObjectLayoutField.tsx

```diff
@@ -179,6 +179,7 @@ export default function ModalAddObjectLayoutField({
 							'there-are-no-fields-for-this-object'
 						)}
 						error={errors.objectFieldName}
+						id="modal-add-object-layout-field"
```

Comment:
Hi @Artartur. Please, rename the id to `modalAddObjectLayoutField`.

Reply:
Done!